AUTHORS

```diff
@@ -0,0 +1,195 @@
+# Authors ordered by first contribution.
+
+## (generated with `git log --encoding=utf-8 --full-history --reverse "--format=format:%at;%an;%ae" | sort -t ';' -u -k 3,3 | sort | awk -F ';' '{ print $2 " <" $3 ">" }'`)
```

Comment:
I was wondering about this too. I did not add one to the bot project, but as @ljharb mention elsewhere in this thread, @mhdawson added one to nodereport in https://github.com/nodejs/nodereport/pull/1 so he added it here too. Question for the TSC I guess?